src/Tasks/WriteCodeFragment.cs

```diff
@@ -44,6 +48,17 @@ public class WriteCodeFragment : TaskExtension
         [Required]
         public string Language { get; set; }
 
+        /// <summary>
+        /// If the projects have ImplictUsings set, then use it's value.
+        /// Anything other than "enable" changes nothing here.
+        /// When it is set to "enable" then do not add the usings that
+        /// this normally writes to the output file.
+        /// This leverages the full power of what was added in .NET 6+
+        /// called global usings of C# 10. Also it leverages the feature
+        /// that existed for a long time in Visual Basic as well.
+        /// </summary>
+        public string ImplicitUsings { get; set; } = "disable";
```

Comment:
A follow-up pr will need to be made to remove ``= "disable";`` here after the .NET SDK starts using it everywhere this task is used.